tests/templates/kuttl/kerberos/20-assert.yaml

```diff
@@ -26,3 +26,11 @@ metadata:
 status:
   readyReplicas: 2
   replicas: 2
+---
```

Comment:
Thanks for adding the test!
I mean it would have been awesome to add that to all operators but that's effort ;)